Sprint-3/alarmclock/alarmclock.js

```diff
@@ -1,5 +1,59 @@
-function setAlarm() {}
+const heading = document.querySelector("#timeRemaining");
+const stopBtn = document.querySelector("#stop");
 
+const headerText = 'Remaining Time:'
+formatAndDisplayTime(totalTimeInSeconds = 0) // displays initial heading with timer values (00:00)
```

Comment:
Aha! This is something interesting to dig into!

In this code:
```js
function formatAndDisplayTime(totalTimeInSeconds){
```

The function `formatAndDisplayTime` is saying "You should pass me one value as an argument. I will give it a name, `totalTimeInSeconds`, for as long as I'm running, and that name will disappear when I finish running". It's a name that only matters to the function itself inside of it.

So if you call `formatAndDisplayTime(0)`, when we jump inside `formatAndDisplayTime` it will assign `0` to `totalTimeInSeconds`, and when `formatAndDisplayTime` finishes, there won't be something called `totalTimeInSeconds` any more.

Everywhere that you have a variable called `totalTimeInSecond`, you could use a completely different name if you wanted to - the name of what you have when you _call_ a function is unrelated to the name you use when you're inside the function.

Reply:
Thank you for clarifying this. It is really helpful